packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/process/error.ts

```diff
@@ -120,15 +120,9 @@ export function handleSchemaErrorResponse({
 }
 
 /**
- * Retries if the response has an error that is not in the known non-retryable errors list
- * Will not retry if the response is successful and is not an error.
+ * For now lets not retry on any error. We have network retries already. s
  */
 export function isRetryable(outboundResponse: OutboundPatientDiscoveryResp | undefined): boolean {
   if (!outboundResponse) return false;
-  return (
-    outboundResponse.operationOutcome?.issue.some(
-      issue =>
-        issue.severity === "error" && issue.code !== "http-error" && issue.code !== "schema-error"
-    ) ?? false
-  );
+  return false;
```

Comment:
IIUIC, this is quite important that should be on the PR description. :)

We're not retrying PDs any longer?

Reply:
It seems you commented on a commit and not the whole PR? What we have before this pr is xcpds being retried for network errors in saml-client.ts, but not for errors within the xml itself. 

This PR was just trying to move Xcpd structure to the same thing as dq and dr. This function is for determining if we do retries for errors within the xml itself, and its just keeping the behavior consistent with what it was before but being more explicit, and its saying we dont do retries right now for the content of the xml response

So just to confirm this isnt actually a change at all from what we are currently doing and we are still retrying xcpds for network errors like before